### PR TITLE
Reader Refresh: ManageV1 - increase the thickness of searchbox to 2px

### LINKED ISSUE #9143: Refreshed Manage v1
Update Manage to fit into the Refresh. See Readersquad p2 post "What’s Needed to Launch Refreshed Manage with Search" for details.

Because it's unlikely that we'll get to the full Refresh of Manage before launch, this issue is MVP version.

manage-1

So, just:
• Change background to white.
• Remove button borders from "Following" and "Follow" links.

That's it! 

If we have more time, we'll move on to v2, which I'll post as a separate issue.

### PR DESCRIPTION
**Pull request by samouri**
This PR aims to finish implementing the changes to the Manage Following page described in #9143.

Specifically, this increases the thickness of the search box to 2px (from 1px).

Screenshot of how it should look:
default

Fixes #9143.

### CODE REVIEW

**Comment by samouri** [comment]
Compare these two versions to test:

- https://wpcalypso.wordpress.com/following/edit
- https://calypso.live/following/edit?branch=add/reader/refresh-following-edit-thicker-searchbox-border

cc @fraying

**Comment by fraying** [comment]
LGTM!

File: client/reader/following-edit/style.scss [hunk 1]
```diff
@@ -44,6 +44,10 @@
 	float: right;
 }
 
+.is-reader-page .search-card.card {
+	box-shadow: 0 0 0 2px rgba(200, 215, 225, 0.5), 0 1px 2px #e9eff3;
```
Comment (bluefuton) [inline client/reader/following-edit/style.scss]: We try not to use custom colors if possible. From the [coding guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md):

> Don't use custom colors, always utilize what _colors.scss provides. If you have to set a color, use lowercase hex values, and shorten them to their smallest expression (like #aaa).

You can also use `lighten` to modify the preset colors, e.g.:

`border-color: lighten( $gray, 10% );`

Comment (samouri) [inline client/reader/following-edit/style.scss]: thanks guys! I took the colors from the Chrome inspector without realizing it was compiled from variables!

fixing!

**Comment by jancavan** [comment]
This is nitpicky and maybe it's just me, but I think the "search result" should also be `2px`? It makes it looks misaligned and not part of the search box:

screenshot 2016-11-21 10 01 34

screenshot 2016-11-21 10 04 44

So this instead:
screenshot 2016-11-21 10 03 38

I also had to update `is-search-result` from `top: 70px;` to `top: 71px;` to achieve this.

**Comment by fraying** [comment]
I love it when you pick nits, @jancavan

**Comment by bluefuton** [comment]
I've bumped up the border on the search result to 2px as per @jancavan's request.

**Comment by samouri** [comment]
Tested:
- [x] search results have 2px borders
- [x] tested search results without refresh

The good news (refresh):
Looks good to me

The bad news (no-refresh):
no-refresh

The best news: [2ba45d5](https://github.com/Automattic/wp-calypso/pull/9460/commits/2ba45d5d2664b66f47fa9d8ee241fac2e5403ef1)

**Comment by jancavan** [comment]
:shipit:

**Pull request merged**